sdk/core/ci.yml

```diff
@@ -28,7 +28,11 @@ pr:
       - sdk/core/
       - eng/
       - common/
-      - rush.json
+      - package.json
```

Comment:
This pr section is dead code now after the move to `js - pullrequest`. You will want to add other trigger paths below in the artifacts if you want to trigger on other files.

Reply:
Good point!